core/Site.php

```diff
@@ -422,6 +422,10 @@ public function getCreatorLogin()
      */
     public static function getIdSitesFromIdSitesString($ids, $_restrictSitesToLogin = false)
     {
+        if (empty($ids)) {
+            return [];
```

Comment:
Didn't this method already always returns an array, even if `$ids` is empty?
If that isn't the case in PHP 8.1, would be good to know why that is the case.
Afaik `array_filter` and `array_unique` should always return an array...

Reply:
Ah. I see. The explode would be the problem in PHP 8.1.